Refine Use Cases and DataLink descriptions in UseCases.tex (#62)
ivoa/HighEnergyObsCoreExt: UseCases.tex

AND (access_format = ’application/x-votable+xml;content=datalink’)
\end{verbatim}

> Then, for each row of the output, we get access to a datalink table (in VOTable) describing associated data linked to the event-list dataset using the ``access\_url'' column value of the response ObsCore table.

Reviewer: Change "datalink" to "DataLink" here.

AND (energy_max >= 1.0e+12)
\end{verbatim}

The query output is a VOTable that follows the DataLink VO standard.

We process this VOTABLE to access to the data:

\begin{enumerate}[(i)]
  \item for each row of the query output, get the ``obs\_id'' and the ``access\_url'' of the DataLink describing the ObsCore dataset entry,

>   \item get the Datalink VOTable showing the datasets associated to this entry

Reviewer: Change "Datalink" to "DataLink" here.

>   \item get the Datalink VOTable showing the datasets associated to this entry

Reviewer 2: ```suggestion
  \item get the DataLink VOTable showing the datasets associated to this entry
```

>   \item for each row of the Datalink VOTable, get the ``content\_qualifier'' and the ``access\_url'' column's value,

Reviewer: Change "Datalink" to "DataLink" here.

>   \item for each row of the Datalink VOTable, get the ``content\_qualifier'' and the ``access\_url'' column's value,

Reviewer 2: ```suggestion
  \item for each row of the DataLink VOTable, get the ``content\_qualifier'' and the ``access\_url'' column's value,
```

  \item download the data associated to each ``access\_url'' value.
\end{enumerate}
% removed pseudo code

Table \ref{tab:datalink1} displays an example of the DataLink response table attached to such an event-list discovery.
The obs\_publisher\_did of the single discovered event-list is repeated in the ID column of the DataLink table.
Mandatory FIELDS service\_def and error\_messsage are omitted because they are empty.

\begin{landscape}
\begin{center}
\begin{longtable}{|p{0.15\linewidth}|p{0.2\linewidth}|p{0.08\linewidth}|p{0.22\linewidth}|p{0.1\linewidth}|p{0.12\linewidth}|p{0.13\linewidth}|}
\hline%\sptablerule
\textbf{ID}  &\textbf{\footnotesize access\_url}  &\textbf{\footnotesize semantics}&\textbf{\footnotesize description} &\textbf{\footnotesize content\_type} &\textbf{\footnotesize content\_length} &\textbf{\footnotesize content\_qualifier}\cr
\hline%\sptablerule

> {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt1.ext1} & {\footnotesize \#this} & {\footnotesize event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize fits/table} & {\footnotesize 1000000} & {\footnotesize event-list} \cr

Reviewer 2: ```suggestion
{\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt1.ext1} & {\footnotesize \#this} & {\footnotesize event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize application/x-fits-gadf} & {\footnotesize 1000000} & {\footnotesize event-list} \cr
```

\hline%\sptablerule

> {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt2.ext2} & {\footnotesize \#calibration} & {\footnotesize Effective AREA of the telescope/instrument associated with the event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize OGIP-fits ?} & {\footnotesize 10000} & {\footnotesize aeff} \cr

Reviewer 2: ```suggestion
{\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt2.ext2} & {\footnotesize \#calibration} & {\footnotesize Effective AREA of the telescope/instrument associated with the event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize application/x-fits-gadf} & {\footnotesize 10000} & {\footnotesize aeff} \cr
```

\hline%\sptablerule

> {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt3.ext3} & {\footnotesize \#calibration} & {\footnotesize Energy dispersion of event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize fits/table} & {\footnotesize 10000} & {\footnotesize edisp} \cr

Reviewer 2: ```suggestion
{\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt3.ext3} & {\footnotesize \#calibration} & {\footnotesize Energy dispersion of event-list}  {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize application/x-fits-gadf} & {\footnotesize 10000} & {\footnotesize edisp} \cr
```

\hline%\sptablerule

> {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt4.ext4} & {\footnotesize \#calibration} & {\footnotesize Point spread function of event-list } {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize image/fits} & {\footnotesize 50000} & {\footnotesize psf} \cr

Reviewer 2: ```suggestion
{\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt4.ext4} & {\footnotesize \#calibration} & {\footnotesize Point spread function of event-list } {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize application/x-fits-gadf} & {\footnotesize 50000} & {\footnotesize psf} \cr
```

\hline%\sptablerule

> {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt5.ext5} & {\footnotesize \#calibration} & {\footnotesize Background rate of  event-list } {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize text/csv} & {\footnotesize 1000} & {\footnotesize bkgrate} \cr

Reviewer 2: ```suggestion
{\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize https://xxx.yyy/zzz/ttt5.ext5} & {\footnotesize \#calibration} & {\footnotesize Background rate of  event-list } {\footnotesize ivo://xxx/yyy/zzz\#ttt} & {\footnotesize application/x-fits-gadf} & {\footnotesize 1000} & {\footnotesize bkgrate} \cr
```
